orbbec_acquire/util.py

```diff
@@ -298,11 +300,10 @@ def start_recording(
 
             depth_data = np.frombuffer(depth_frame.get_data(), dtype=np.uint16)
             depth_data = depth_data.reshape((height, width))
-            depth_data = depth_data.astype(np.float32) * scale
-            depth_data = np.where(
-                (depth_data > MIN_DEPTH) & (depth_data < MAX_DEPTH), depth_data, 0
-            )
-            depth_data = depth_data.astype(np.uint16)
+            depth_data = (depth_data * scale).astype(np.uint16)
+            # depth_data = np.where(
+            #     (depth_data > MIN_DEPTH) & (depth_data < MAX_DEPTH), depth_data, 0
+            # )
```

Comment:
I prefer to keep this since that's how the example in pyorbbecsdk is implemented. If you want to change this, please explain why.

Reply:
I want to change this because replacing the acquired depth values beyond the `MIN_DEPTH` and `MAX_DEPTH` range is throwing away data, which I don't think is a good idea at this stage of the pipeline.